src/templates/upf.json.j2

```diff
@@ -0,0 +1,47 @@
+{
+  "access": {
+    "ifname": "access"
+  },
+  "core": {
+    "ifname": "core"
+  },
+  "cpiface": {
+    "dnn": "internet",
+    "enable_ue_ip_alloc": false,
+    "hostname": "{{ upf_hostname }}",
+    "http_port": "8080",
+    "ue_ip_pool": "172.250.0.0/16"
```

Comment:
Don't we want that to be configurable?

Reply:
This won't end up being used since we set enable_ue_ip_alloc to False. I simply removed the key/value from the config file now